algorithm/main.go

```diff
@@ -0,0 +1,62 @@
+// Package algorithm provides functionalities to interact crystal
+package algorithm
+
+import (
+	"fmt"
+
+	"github.com/SneaksAndData/esd-services-api-client-go/shared/http"
+)
+
+// Service encapsulates the HTTP client and URLs needed to interact with the algorithm service.
+type Service struct {
+	httpClient   *http.Client
+	schedulerURL string
+	receiverURL  string
+	apiVersion   string
+}
+
+// Payload defines the structure of the request body for creating algorithm runs.
+type Payload struct {
+	AlgorithmParameters interface{}
+	AlgorithmName       string
+	CustomConfiguration interface{}
```

Comment:
hmm for this one I think you can define a struct as fields for this are well-known? Or is there a reason you opted for `any value`?

Reply:
Added CustomConfiguration struct